Octokit/Http/InMemoryCredentialStore.cs

```diff
@@ -15,7 +15,7 @@ public InMemoryCredentialStore(Credentials credentials)
 
         public Task<Credentials> GetCredentials()
         {
-            return Task.Factory.StartNew(() => _credentials);
+            return Task.FromResult(_credentials);
```

Comment:
No objections. In fact, I meant to make this change everywhere. We should never be using `StartNew` just to wrap an object in a task.